src/main/java/com/sarality/db/query/SimpleQueryBuilder.java

```diff
@@ -16,6 +16,8 @@
 
   private final LogicalOperator operator;
   private final List<QueryClause> clauseList = new ArrayList<>();
+  private final List<AggregateMeasure> measuresList = new ArrayList<>();
```

Comment:
I didnt use a Map here since in SQL you can have two aggregate functions on the same column. If I made the column the key I'd only be able to support one function. So I created a tuple for AggregateMeasure and created a list of these measures instead of the tableColumnList I had earlier.